src/main/java/pageobject/HomeActionPage.java

```diff
@@ -0,0 +1,18 @@
+package pageobject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomeActionPage extends AbstractPage {
```

Comment:
Do not create a new class if this is the same page. U already have HomePage
Same for LoginActionsPage and other if u have

The main feature of page object that u have one class for selected screen with all described locators and methods.
In case u create LoginActionsPage u duplicate code.

Rename all alternate methods, try to define what method does in name
e.g.
public HomePage clickLoginButton()
public HomeActionPage clickLoginButtonWithActionAndHighlight()

Reply:
Done